cmd/placement-translator/main.go

```diff
@@ -204,8 +193,8 @@ func main() {
 	mbwsPreInformer := rootInformerFactory.Tenancy().V1alpha1().Workspaces()
 	var _ tenancyv1a1informers.WorkspaceInformer = mbwsPreInformer
```

Comment:
This line should have been deleted months ago, it was just a temporary thing to help write the code.

Reply:
Removed